### PR TITLE
D8CORE-855 Move toolbar user tab to far right

### PR DESCRIPTION
**Pull request by pookmish**
# READY FOR REVIEW

# Summary
- Moved the "User" tab to the right of the toolbar.

# Need Review By (Date)
- 12/11

# Urgency
- low

# Steps to Test
1. checkout this branch
1. clear cache
1. verify the user tab in the toolbar is at the right.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: modules/stanford_profile_styles/stanford_profile_styles.module [hunk 1]
```diff
@@ -1,12 +1,20 @@
...
  */
-function stanford_profile_styles_page_attachments(array &$page) {
+
+/**
+ * Implements hook_page_attachments().
+ */
+/**
+ * Implements hook_page_attachments_alter().
+ */
+function stanford_profile_styles_page_attachments_alter(array &$attachments) {
```
Comment (sherakama) [inline modules/stanford_profile_styles/stanford_profile_styles.module]: Why did this change to hook_alter?

Comment (pookmish) [reply modules/stanford_profile_styles/stanford_profile_styles.module]: It didn't actually work. Once I changed to an alter the css got added. Same for the Stanford page css

Comment (sherakama) [reply modules/stanford_profile_styles/stanford_profile_styles.module]: It's working as hook_page_attachments for me. I see the stylesheets in the head. Can you try again?

Comment (pookmish) [reply modules/stanford_profile_styles/stanford_profile_styles.module]: hmm yeah now it is working again. wth. :man_shrugging: 
I'll try to fix up the tests now. something in caravan is loading `8.8.0-alpha1` of core for some reason.

**Comment by pookmish** [comment]
Passed tests. now good for full review

**Comment by sherakama** [comment]
Cool, so the hook works for you now as well?

**Pull request merged**